kitsune/kpi/cron.py

```diff
@@ -632,7 +632,7 @@ def calculate_csat_metrics():
             timeout=300)
 
         results = json.loads(response.content)
-        total_pages = results['total_pages']
+        total_pages = results['total_pages'] if 'total_pages' in results else 1
```

Comment:
I think its better written as
``results.get('total_pages', 1)``
What do you think?